client/components/button/block.jsx

```diff
@@ -0,0 +1,51 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { omit } from 'lodash';
+import PropTypes from 'prop-types';
+
+class BlockButton extends Component {
```

Comment:
Given that this is a new component, I would expect this under `client/components/block-button/index.jsx` instead. (Searching is much easier if we're consistent with this.)

Do folks think otherwise?

Reply:
Would a better name for this be something like `FocusableItem` or `Focusable` since it does look like we can take arbitrary children

Reply:
I had the same thought when naming it, I was thinking block like "could contain block elements" which is definitely not clear. I like `Focusable` 👍